meta-iot2000-example/recipes-node/node-red-node-intel-gpio/node-red-node-intel-gpio_0.0.5.bb

```diff
@@ -4,8 +4,8 @@
 #
 # WARNING: the following LICENSE and LIC_FILES_CHKSUM values are best guesses - it is
 # your responsibility to verify that the values are complete and correct.
-LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://${IOT2000_MIT_LICENSE};md5=838c366f69b72c5df05c96dff79b35f2"
+LICENSE = "Unknown"
```

Comment:
Hm, shouldn't the licensing be clarified and explicitly stated? Or what is the reason for this?